scripts/typescript_checks.py

```diff
@@ -77,7 +77,7 @@ def compile_and_check_typescript(config_path):
     process = subprocess.Popen(cmd, stdout=subprocess.PIPE)
     error_messages = []
     for line in iter(process.stdout.readline, ''):
-        if not line.startswith('node_modules'):
+        if not line.decode('utf-8').startswith('node_modules'):
```

Comment:
With this change we can add a directory in include array, I have added one for example(Can help us reduce size of file object)